internal/controller/pipelinerollout/pipelinerollout_progressive.go

```diff
@@ -386,7 +407,7 @@ func (r *PipelineRolloutReconciler) ProcessPromotedChildPreRecycle(
 
 	numaLogger.Debug("completed pre-recycle processing of promoted pipeline")
 	return nil
-}
+}*/
```

Comment:
Didn't need these "PreRecycle" functions anymore since the recycling logic is now in the Recycle function